src/extremeweatherbench/utils.py

```diff
@@ -303,3 +299,40 @@ def process_dataarray_for_output(da_list: List[Optional[xr.DataArray]]):
         output_da = output_da.isel(lead_time=slice(None, None, -1))
     output_da = expand_lead_times_to_6_hourly(output_da)
     return output_da
+
+
+def center_forecast_on_time(da: xr.DataArray, time: pd.Timestamp, hours: int):
+    """Center a forecast DataArray on a given time with a given range in hours.
+    Args:
+        da: The forecast DataArray to center.
+        time: The time to center the forecast on.
+        hours: The number of hours to include in the centered forecast.
+    """
+    time_range = pd.date_range(
+        end=pd.to_datetime(time) + pd.Timedelta(hours=48), periods=97, freq="h"
+    )
```

Comment:
You accept an `hours` argument but hard-code **hours** and **periods** here... should generalize to use the arguments from the user.